Test in kitchen-dokken

tas50 opened the pull request:
Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Test in kitchen-dokken

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Test with Local Delivery and not Rake

Signed-off-by: Tim Smith <tsmith@chef.io>

lamont-granquist commented:
:-1: cookstyle

i'm probably going to pull this into chef-client core for 13.0 or 13.1 so you should probably stop trying to fight with it.

tas50 committed:
Ensure chef 12.5 compatibility

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Require Chef 12.7+ in order to be Chef 13 compatible

action_class.class_eval doesn't work on Chef 13, but is required for
Chef < 12.7.

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Use a local delivery config and fix the specs

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 commented:
There's no more cookstyle here and this passes / aligns us with other cookbooks so I'm merging this one
